packages/app/lib/debug.ts

```diff
@@ -0,0 +1,23 @@
+import { useDebugStore } from '@tloncorp/shared';
+
+import storage from './storage';
+
+const DEBUG_STORAGE_KEY = 'debug';
+
+storage
+  .load({
+    key: DEBUG_STORAGE_KEY,
+  })
+  .then((enabled) => {
+    useDebugStore.getState().toggle(enabled);
+  });
+
+export const toggleDebug = async (enabled: boolean) => {
```

Comment:
nit, `setDebug` may be clearer since you're passing the value you want and not flipping curr